improver_tests/utilities/copy_metadata/test_CopyMetadata.py

```diff
@@ -68,11 +70,46 @@ def test_copy_attributes_single_input():
         [0], attributes={"attribA": "tempA", "attribB": "tempB", "attribC": "tempC"}
     )
 
-    plugin = CopyAttributes(attributes)
+    plugin = CopyMetadata(attributes)
     result = plugin.process(cube0, template_cube)
     assert type(result) is Cube
     assert result.attributes["attribA"] == "tempA"
     assert result.attributes["attribB"] == "tempB"
     assert result.attributes["attribD"] == "valueD"
     assert "attribC" not in result.attributes
     assert id(result) == id(cube0)
+
+
+@pytest.mark.parametrize("cubelist", [True, False])
```

Comment:
I think it would be helpful if you could make this clearer in some way - whether this is splitting into two tests or just explaining within that test?

Reply:
I've updated the docstring of the test to clarify what its doing as splitting the test was producing lots of repeated code.